src/common/utils/api-id.config.ts

```diff
@@ -14,10 +14,13 @@ export const APIID = {
     PRIVILEGE_BYPRIVILEGEID: 'api.privilegebyPrivilegeId.get',
     PRIVILEGE_CREATE: 'api.privilege.create',
     PRIVILEGE_DELETE: 'api.privilege.delete',
-    COHORT_CREATE:"api.cohort.create",
-    COHORT_LIST:"api.cohort.list",
-    COHORT_READ:"api.cohort.read",
-    COHORT_UPDATE:"api.cohort.update",
-    COHORT_DELETE:"api.cohort.delete",
-    FIELDVALUES_CREATE:"api.fieldValues.create"
+    USERROLE__CREATE: "api.userRole.create",
```

Comment:
@poojakarma why double underscore is used in USERROLE__CREATE?

Reply:
Resolved